docs/source/api/auth.md

```diff
@@ -24,10 +24,12 @@
 
 ```{eval-rst}
 .. autoconfigurable:: PAMAuthenticator
+   :members:
 ```
 
 ### {class}`DummyAuthenticator`
 
 ```{eval-rst}
 .. autoconfigurable:: DummyAuthenticator
+   :members:
```

Comment:
Previews:

This PR: https://jupyterhub--4253.org.readthedocs.build/en/4253/api/auth.html#dummyauthenticator
Main branch: https://jupyterhub.readthedocs.io/en/latest/api/auth.html#dummyauthenticator